RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -30,16 +30,7 @@ namespace lst {
     //debug variables
     float* zOut;
     float* rtOut;
-    float* deltaPhiPos;
```

Comment:
was the `CUT_VALUE_DEBUG` case tested locally?

Reply:
Yes, and it compiles ok. The change was to remove variables that we were not setting in the code (so the arrays were filled with the random (garbage) initialization values).